packages/gatsby-source-drupal/src/__tests__/fixtures/paragraph-v1.json

```diff
@@ -0,0 +1,395 @@
+[
+    {
+        "jsonapi": {
+            "version": "1.0",
+            "meta": {
+                "links": {
+                    "self": {
+                        "href": "http://jsonapi.org/format/1.0/"
+                    }
+                }
+            }
+        },
+        "data": {
+            "type": "node--page",
+            "id": "6a5db0d4-4a0f-4a4b-9048-7d26e29fdacb",
+            "links": {
+                "self": {
+                    "href":
+"https://dev-drupal-bug-reproduction.pantheonsite.io/jsonapi/node/page/6a5db0d4-4a0f-4a4b-9048-7d26e29fdacb?resourceVersion=id%3A13"
```

Comment:
Did you want the fixtures to have this URL in place?

Reply:
Meh it doesn't matter much. We discard that from the JSON:API when processing it anyways.